submissions/sprint_b/app.py

```diff
@@ -0,0 +1,36 @@
+from sprint_b import app
+import json
+from helpers import EpithetGenerator, Vocabulary
+
+path = ('/Users/ethansternke/Documents/Kenzie/back-end'
+        '/backend-epithet-generator/resources/data.json')
+
+
+@app.route('/')
+def generate_epithet():
+    e = EpithetGenerator()
+    epithet = e.generate_epithet(path)
+    response = {"epithet": epithet}
+    return json.dumps(response)
+
+
+@app.route('/vocabulary')
+def vocabulary():
+    v = Vocabulary()
+    vocabulary = v.from_file(path)
+    response = {"vocabulary": vocabulary}
+    return json.dumps(response)
+
+
+@app.route('/epithets/<quantity>')
```

Comment:
You can avoid typecasting int(quantity) yourself using variable rules in your route declaration.
http://flask.pocoo.org/docs/1.0/quickstart/#variable-rules